api_reference/source/conf.py

```diff
@@ -1,53 +1,197 @@
+    member_name = name.split('.')[-1]  # Get the last part of the name
+
+    # Check if the module is in the include list
+    if module_name in INCLUDE_MEMBERS:
+        if class_name in INCLUDE_MEMBERS[module_name]:
+            # Check if the member is in the include list for this class
+            return member_name not in INCLUDE_MEMBERS[module_name][class_name]
+        # Skip all other classes not specifically included
+        return True
+    # Skip all other modules not specifically included
+    return True
+
+def filter_rst(app, docname, source):
+    if source:
+        processed = []
+        in_automodule = False
+        
+        # Iterate over each line in the source
+        for line in source[0].split('\n'):
+            # Check for the start of the automodule block
+            if '.. automodule::' in line:
+                in_automodule = True
+
+            # If line is empty and we are in the automodule block, continue to consider it inside
+            if in_automodule and line.strip() == '':
+                processed.append(line)
+                continue  # Skip the reset of in_automodule until out of relevant content
+
+            # Set in_automodule to false once out of the block
+            if in_automodule and line.strip() and not line.startswith('   '):
+                in_automodule = False
+
+            # Replace the unwanted text outside automodule blocks
+            if not in_automodule:
+                if 'Submodules' in line:
+                    continue 
+                if 'Module contents' in line:
+                    continue 
+                if ' package' in line:
+                    line = line.replace(' package', '')  # Remove ' package'
+                if ' module' in line:
+                    line = line.replace(' module', '')  # Remove ' module'
+
+            # Append potentially modified line to new output
+            processed.append(line)
+
+        # Join the modified lines back into a single string
+        source[0] = '\n'.join(processed)
 
 def setup(app):
-    if len(app.tags.tags) > 0:
-        app.connect("autodoc-skip-member", include_only_tagged)
+    # if len(app.tags.tags) > 0:
+    #     app.connect("autodoc-skip-member", include_only_tagged)
+    # app.connect("autodoc-skip-member", skip_member)
 
+    # Remove unnecessary headers
+    app.connect('source-read', filter_rst)
 
-# Configuration file for the Sphinx documentation builder.
-#
-# For the full list of built-in configuration values, see the documentation:
-# https://www.sphinx-doc.org/en/master/usage/configuration.html
 
 # -- Project information -----------------------------------------------------
-# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-project = "arize-phoenix"
+project = "Phoenix API"
```

Comment:
```suggestion
project = "Phoenix API Reference"
```

Just because we also have an actual "API"